design/one-pager-resource-reclaim-policy.md

```diff
@@ -0,0 +1,63 @@
+# Resource Reclaim Policy
+* Owner: Daniel Mangum (@hasheddan)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Terminology
+
+* **`External Resource`**: an infrastructure resource that runs outside of Crossplane (i.e. an S3 Bucket on AWS).
+* **`Concrete Resource`**: a Kubernetes resource that is responsible for managing an external resource (recieves configuration details from the `ResourceClass` and `ResourceClaim`). It is of type `Resource` but will be referred to as `Concrete Resource` consistently here to avoid confusion.
+* **`ResourceClass`**: a Kubernetes resource that contains implementation details specific to a certain environment or deployment, and policies related to a kind of resource.
+* **`ResourceClaim`**: a Kubernetes resource that captures the desired configuration of a resource from the perspective of a workload or application.
+
+## Background
```

Comment:
I think it could also be useful to provide background on the `PersistentVolume` and `StorageClass` reclaim policy fields that inspired our design and how they work, as well as the current Crossplane equivalent to those fields (i.e. the garbage collection of resources when their claims are deleted due to owner references).

Reply:
Ah - I see you detail the Crossplane owner reference pattern under workflow.

Reply:
Yep! I also think it would be useful to provide some more context around PV's as that is likely the only place that most users will have encountered reclaim policies before.